src/Components/Cards/PlaylistCard.jsx

```diff
@@ -0,0 +1,44 @@
+import { Link } from "react-router-dom";
+import { useState } from "react";
+import { useAuth, useAxiosCalls } from "../../Context";
+
+export const PlaylistCard = ({ list }) => {
+  const { removePlayListFromServer } = useAxiosCalls();
+  const {
+    auth: { token },
+  } = useAuth();
+
+  const [trash, showTrash] = useState(true);
+
+  const playlistConfig = {
+    url: "/api/user/playlists",
+    headers: { headers: { authorization: token } },
+    playlistId: list._id,
```

Comment:
Instead of passing token in each request, you can use axios interceptors to get the token and append it to the request header. This will remove code repeatition.

Reply:
I didn't know about Axios inceptors, will fix it in future.